message_command.py

```diff
@@ -191,36 +192,42 @@ async def util_command(self, _):
         user_message = self.message[1:]
         commands = user_message.split()
 
-        if commands[0] == "lol":
+        command_name = commands[0].lower()
+
+        if command_name == "lol":
             await MessageCommands.LOL_COUNTER.output(self.channel, self.member_id)
             return
 
-        if commands[0] == "typo":
+        if command_name == "typo":
             await self.channel.send(
                 MessageCommands.TYPO_COUNTER.call(self.member_id, self.member_name)
             )
             return
 
-        if commands[0].lower() in ["jd" or "judge"]:
+        if command_name in ["jd" or "judge"]:
             await MessageCommands.MANUAL_JUDGE.call(commands, self.channel)
             return
 
-        if commands[0].lower() == "partyichiyo":
+        if command_name == "partyichiyo":
             await MessageCommands.PARTY_ICHIYO.change_command(commands, self.channel)
             return
 
-        if commands[0].lower() == "kaere":
+        if command_name == "kaere":
             await MessageCommands.KAERE.command_controller(commands, self.member_name)
             return
 
-        if commands[0].lower() in ["number", "zinnkou", "zinkou", "population"]:
+        if command_name in ["number", "zinnkou", "zinkou", "population"]:
             await number(self.channel)
             return
 
-        if commands[0].lower() == "role":
+        if command_name == "role":
             await role(commands, self.channel, self.member_name)
             return
 
+        if command_name in ["debug", "d"]:
+            await debug_on_message(commands, respond_channel=self.channel)
+            return
```

Comment:
```suggestion
        if command_name in ["debug", "d"] and len(commands)  >= 2:
            await debug_on_message(commands[1], respond_channel=self.channel)
            return
```
これじゃ駄目なんですか...
list渡す意味が見えません...

Reply:
最新のコミットでやった（時間差）